bin/ebs-autoscale

```diff
@@ -100,15 +100,17 @@ add_space () {
     return 0
   fi
 
-  local curr_size=$(df -BG ${MOUNTPOINT} | grep ${MOUNTPOINT} | awk '{print $2} ' | cut -d'G' -f1)
-  if [ "${curr_size}" -lt "$MAX_LOGICAL_VOLUME_SIZE" ]; then
+  local curr_autoscaled_size=$(cat /var/run/ebs_autoscale_created_size)
+  if [ "${curr_autoscaled_size}" -lt "$MAX_LOGICAL_VOLUME_SIZE" ]; then
     local vol_size=$(calc_new_size ${num_devices})
     loginfo "Extending logical volume ${MOUNTPOINT} by ${vol_size}GB"
 
     DEVICE=$(${BASEDIR}/create-ebs-volume --size ${vol_size} --max-attached-volumes ${MAX_EBS_VOLUME_COUNT})
 
     exit_status=$?
     if [ $exit_status -eq 0 ]; then
+      # Update the total autoscaled size
+      echo $((total_autoscaled_size + vol_size)) > /var/run/ebs_autoscale_created_size
```

Comment:
where is `total_autoscaled_size` defined?

Reply:
I made a mistake in renaming the variable. I will fix it right away.